changelog

```diff
@@ -2,6 +2,10 @@ MSAL Wiki : https://github.com/AzureAD/microsoft-authentication-library-for-andr
 
 vNext
 ----------
+
+Version 6.0.0
+----------
+- [PATCH] Update common @21.0.0
 - [MAJOR] Update minSDK to 21
 
```

Comment:
Can you please also include this:
```
Version 5.10.2
----------
- [PATCH] Update common @20.1.1
```
Looks like I didn't merge in the previous release branch for some reason, sorry

Reply:
Done